hook.py

```diff
@@ -1,19 +1,20 @@
 from app.objects.c_obfuscator import Obfuscator
-from plugins.stockpile.app.http import HTTP
+from plugins.stockpile.app.contact.http import HTTP
 from plugins.stockpile.app.stockpile_svc import StockpileService
+from plugins.stockpile.app.contact.gist import GIST
 
 name = 'Stockpile'
 description = 'A stockpile of abilities, adversaries, payloads and planners'
 address = None
 
 
 async def enable(services):
-    file_svc = services.get('file_svc')
-    await file_svc.add_special_payload('mission.go', StockpileService(file_svc).dynamically_compile)
-
-    data_svc = services.get('data_svc')
-    await data_svc.load_data(directory='plugins/stockpile/data')
-    await services.get('contact_svc').register(HTTP(services))
-    await services.get('data_svc').store(
+    stockpile_svc = StockpileService(services)
+    await stockpile_svc.file_svc.add_special_payload('mission.go', stockpile_svc.dynamically_compile)
+    await stockpile_svc.data_svc.load_data(directory='plugins/stockpile/data')
+    c2_configs = await stockpile_svc.load_c2_config(directory='plugins/stockpile/data/contact')
+    await stockpile_svc.contact_svc.register(HTTP(services))
+    await stockpile_svc.contact_svc.register(GIST(services, c2_configs['GIST']), active=True)
```

Comment:
should active=True actually be the 'enabled' field from the C2 YML, instead?

Reply:
Yes, I made a stupid - updating now.

Reply:
active also refers to it's type - I'm going to iterate on this quickly